File: JAVA/Array/FindSecondLargest.java
public class FindSecondLargest {
    int[] nums = {2,3,5,7,8};
    public int findSecondLargest(int[] nums) {
        int max = 0;
        int secondMax = 0;
        for(int num : nums){
            if(max < num){
                secondMax = max; 
                max = num;
            }
            else if(secondMax < num && num != max){
                secondMax = num;

            }
        }
        return secondMax;
    }
    public static void main(String[] args) {
        FindSecondLargest findSecondLargest = new FindSecondLargest();  
        System.out.println(findSecondLargest.findSecondLargest(findSecondLargest.nums));
    }

}

File: JAVASCRIPT/FindSecondLargest.js

function FindSecondLargest(arr) {
    let max = 0;
    let SecondMax = 0;
    for (let i = 0; i < arr.length; i++) {
        if(max < arr[i]){
            SecondMax = max;
            max = arr[i];
        }
        else if(SecondMax < arr[i] && arr[i] < max){
            SecondMax = arr[i];
        }
    }
    return SecondMax;
}

let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
console.log(FindSecondLargest(arr));